zerver/openapi/zulip.yaml

```diff
@@ -6732,6 +6754,8 @@ paths:
 
                           Only relevant to clients with configuration UI for choosing an emojiset;
                           the currently selected emojiset is available in the `emojiset` key.
+                          See [update-display-settings](/api/update-display-settings)
+                          for details on display settings.
```

Comment:
Please add a blank line before your new paragraphs, for consistency.

Reply:
Also, we name endpoints in these docs via e.g. `POST /...`, not the URL chunks, if we reference them.  I think cleanest though would be to just see "See the [API documentation for changing display settings](/api/update-display-settings)".

Also, you should reverse the order of the two commits, because the first commit has broken links without the second merged.

Reply:
Made these changes, I have put the links in following pattern:
`[PATCH /update-display-settings](/api/update-display-settings)`

Reply:
Corrected the commit order too.